src/features/compensations/components/AllCompensations/components/AllCompensationsActions/AllCompensationsActions.tsx

```diff
@@ -0,0 +1,14 @@
+import { AllCompensationsFilter } from './components/AllCompensationsFilter/AllCompensationsFilter';
+import { DatePickerAllCompensations } from './components/DateAllCompensations/DatePickerAllCompensations';
+
+// TODO add SearchBarAll
+export function AllCompensationsActions() {
+  return (
+    <div className="compensation-actions" data-cy="compensation-actions">
+      <div className="compensation-actions--div--left" />
```

Comment:
adding the tag name to the class name is not a good practice, because if you change the tag in the markup, you will have to change the class name as well